dashboard/app/models/pd/district_report.rb

```diff
@@ -0,0 +1,59 @@
+module Pd
+  class DistrictReport
```

Comment:
I used this pattern a lot. Do you generally prefer the class name on a single line vs. the module?

The advantage of explicitly declaring the module is the context for other class references?:
```ruby
module Pd
  class DistrictReport
    def foo
      Workshop.new #this is a Pd::Workshop
    end
  end
end
```

vs.
```ruby
class Pd::DistrictReport
  def foo
    ::Pd::Workshop.new # simply Workshop.new would create a ::Workshop
  end
end
```